src/store/modules/order/actions.ts

```diff
@@ -37,5 +37,8 @@ const actions: ActionTree<OrderState, RootState> = {
   updatedOrderListItems({ commit }, orderListItems){
     commit(types.ORDER_LIST_ITEMS_UPDATED, orderListItems)
   },
+  clearOrderList({ commit }){
+    commit(types.ORDER_LIST_CLEARED);
```

Comment:
I think we could use ORDER_LIST_UPDATED and pass empty list items and original

Reply:
Made the requested change.